src/SAWScript/Interpreter.hs

```diff
@@ -1842,6 +1842,15 @@ primitives = Map.fromList
     (pureVal print_goal)
     Current
     [ "Print the current goal that a proof script is attempting to prove." ]
+  , prim "print_goal_inline"   "[Int] -> ProofScript ()"
+    (pureVal print_goal_inline)
+    Current
+    [ "Print the current goal that a proof script is attempting to prove,"
+    , "without generating `let` bindings for the provided indices. For"
+    , "example, `print_goal_inline [1,9,3]` will print the goal without"
+    , "inlining the variables that would otherwise be abstracted as `x@1`,"
+    , " `x@9`, and `x@3`."
```

Comment:
The exact choice of numbers that get used in the pretty-printed output for SAWCore terms is quite sensitive to minor changes in the SAWCore. As such, it would be pretty fragile to rely on this in scripts. I think it is still worth having this as a debugging aid, but it would be worth adding a disclaimer to the documentation here about its fragility.

Reply:
I agree, and will be submitting another PR soon that addresses that sensitivity, but you're right that this warrants a disclaimer.